buildSrc/build.gradle.kts

```diff
@@ -4,11 +4,13 @@ plugins {
 
 repositories {
     jcenter()
+    gradlePluginPortal()
 }
 
 dependencies {
     implementation("org.springframework.boot:spring-boot-gradle-plugin:2.4.2")
     implementation("org.cqfn.diktat:diktat-gradle-plugin:0.4.0")
     implementation("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.15.0")
     implementation("org.ajoberstar.reckon:reckon-gradle:0.13.0")
+    implementation("org.jetbrains.kotlin.plugin.allopen:org.jetbrains.kotlin.plugin.allopen.gradle.plugin:1.3.71")
```

Comment:
Why 1.3.71?

Reply:
This one is on gradle plugins portal and it is the latest version there

Reply:
https://plugins.gradle.org/m2/org/jetbrains/kotlin/kotlin-allopen/1.4.21/

Reply:
I guess what you found is a plugin marker; for dependencies on actual plugin jar you should use artifact coordinates that are in buildscript syntax in examples: https://plugins.gradle.org/plugin/org.jetbrains.kotlin.plugin.allopen